src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -173,7 +176,9 @@ export const SueReportLocation = ({
       <Wrapper style={styles.noPaddingTop}>
         <Input
           name="houseNumber"
-          label={texts.sue.report.houseNumber}
+          label={`${texts.sue.report.houseNumber} ${
+            requiredInputs?.includes('houseNumber') ? '*' : ''
```

Comment:
what does this stand for:` ? '*' : ''`

Reply:
If `requiredInputs` includes the `houseNumber` value, we add `*` to the label. Because this field must be filled. If it is not required, we leave it blank.